### PR TITLE
A11y examples

### PR DESCRIPTION
**Pull request by evanSe**
[skip changelog]

### Context
new a11y demos were not consistent with our existing
- kebab case class/id
- make CSS prettier
- replace svg
- add 3 additional links

### How has this been tested?
ran locally

### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [X] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [X] I have signed the [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1)
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Comment by jansiegel** [comment]
Not to extend the scope of this PR too much, but maybe it could fix the problems described here: https://github.com/handsontable/dev-handsontable/issues/1514?

**Review by budnix — CHANGES_REQUESTED** [review]
While playing a demo, I spotted that the date-type editor behaves buggy. I found that there is a missing `dateFormat` option.

https://github.com/handsontable/handsontable/pull/10578/files#diff-3508ddba73419ba9f798353a01e9eadf9b2ecdc8fee6d854f1b5e3208ff16a5cR37-R41

should be
```js
    {
      data: "sellDate",
      type: "date",
      dateFormat: "D.MM.YYYY",
      allowInvalid: false,
    },
```
The changes apply to all frameworks

Another thing is that the progress bar renderers do not fit in the cell always
Zrzut ekranu 2023-11-13 o 13 01 13

Enabling or disabling the "Enable cells virtualization" option freezes the browser. The table is not usable. I'd recommend changing the dataset size to ~500 rows or less.

File: examples/next/docs/js/accessibility/main.js [hunk 1]
```diff
@@ -72,19 +53,16 @@ const hotOptions = {
...
+  navigableHeaders: true, // New accessibility feature
+  disableTabNavigation: false, // New accessibility feature
   autoWrapRow: true,
   autoWrapCol: true,
   multiColumnSorting: true,
   filters: true,
   rowHeaders: true,
   manualRowMove: true,
-  rowHeaders: true,
   nestedRows: true,
```
Comment (budnix) [inline examples/next/docs/js/accessibility/main.js:64]:
```suggestion
```
There is no more nested dataset used.

File: examples/next/docs/react/accessibility/src/App.tsx [hunk 2]
```diff
@@ -37,7 +32,6 @@ const hotOptions = {
   manualRowMove: true,
   nestedRows: true,
```
Comment (budnix) [inline examples/next/docs/react/accessibility/src/App.tsx:33]:
```suggestion
```

**Review by budnix — CHANGES_REQUESTED** [review]

**Review by budnix — APPROVED** [review]

**Review by budnix — APPROVED** [review]

**Pull request merged**